orix/tests/quaternion/test_orientation.py

```diff
@@ -354,6 +356,36 @@ def test_get_distance_matrix_equal_explicit_calculation(self, symmetry):
         angle2 = p12.angle.min(axis=(0, 2, 3, 5))
         assert np.allclose(angle1, angle2)
 
+    def test_from_align_vectors(self):
+        TETRAGONAL_LATTICE = Lattice(0.5, 0.5, 1, 90, 90, 90)
+        TETRAGONAL_PHASE = Phase(
+            point_group="4", structure=Structure(lattice=TETRAGONAL_LATTICE)
+        )
+        a = Miller([[2, -1, 0], [0, 0, 1]], phase=TETRAGONAL_PHASE)
+        b = Miller([[3, 1, 0], [-1, 3, 0]], phase=TETRAGONAL_PHASE)
+        ori = Misorientation.from_align_vectors(a, b)
+        assert type(ori) == Misorientation
+        assert ori.symmetry == (
+            TETRAGONAL_PHASE.point_group,
+            TETRAGONAL_PHASE.point_group,
+        )
```

Comment:
You only use the lattice once, and there is only one phase in this test, so no need to add the crystal system to the name. These variables were upper case in other test files because they were global variables within the files, which is a convention.

```suggestion
        phase = Phase(
            point_group="4", structure=Structure(lattice=Lattice(0.5, 0.5, 1, 90, 90, 90))
        )
        a = Miller([[2, -1, 0], [0, 0, 1]], phase=phase)
        b = Miller([[3, 1, 0], [-1, 3, 0]], phase=phase)
        ori = Misorientation.from_align_vectors(a, b)
        assert type(ori) == Misorientation
        assert ori.symmetry == (phase.point_group,) * 2
```